element/index.js

```diff
@@ -32,9 +32,34 @@ export { render };
 export { Component };
 
 /**
+ * Creates a copy of an element with extended props.
+ *
+ * @param  {WPElement} element Element
+ * @param  {?Object}   props   Props to apply to cloned element
+ * @return {WPElement}         Cloned element
+ */
+export { cloneElement };
```

Comment:
I could probably be convinced. Needs to be a decision made with future compatibility in mind, related to some of the "reason to be" outlined in the docs:

https://github.com/WordPress/gutenberg/tree/master/element#element